recipes-tweaks/busybox/busybox_append.inc

```diff
@@ -3,12 +3,11 @@
 
 # Released under the MIT license (see COPYING.MIT for the terms)
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
lets not intoduce another .inc to maintain, having a bbappend is simple enough here.

Reply:
k, was thinking of non-master oe-core, but I guess that's not relevant, really.

Reply:
yes eventually we will branch when sumo( 2.5) branches out, for now its just master